app/controllers/stripe-setup/vat-number-company-number/company-number/get.controller.js

```diff
@@ -1,27 +1,8 @@
 'use strict'
 
-// NPM Dependencies
-const lodash = require('lodash')
-
 // Local dependencies
 const { response } = require('../../../../utils/response')
-const { stripeSetup } = require('../../../../paths')
 
 module.exports = (req, res) => {
-  if (lodash.isEmpty(lodash.get(req, 'session.pageData.stripeSetup.vatNumberData.vatNumber'))) {
-    return res.redirect(303, stripeSetup.vatNumberCompanyNumber)
-  }
-
-  let pageData = lodash.get(req, 'session.pageData.stripeSetup.companyNumberData')
-  if (pageData) {
-    delete req.session.pageData.stripeSetup.companyNumberData
-  } else {
-    pageData = {
-      errors: {},
-      companyNumberDeclaration: '',
-      companyNumber: ''
-    }
-  }
-
-  return response(req, res, 'stripe-setup/vat-number-company-number/company-number', pageData)
+  return response(req, res, 'stripe-setup/vat-number-company-number/company-number')
```

Comment:
Minor point - Seeing as we are splitting VAT and Company number - is it worth changing the URL to: 
`stripe-setup/company-number`?

Reply:
yes, planning to change it separately as updating here could make the PR big